src/tests/simulation/src/slt.rs

```diff
@@ -29,9 +32,20 @@ fn is_create_table_as(sql: &str) -> bool {
     parts.len() >= 4 && parts[0] == "create" && parts[1] == "table" && parts[3] == "as"
 }
 
-#[derive(PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq)]
 enum SqlCmd {
-    Create { is_create_table_as: bool },
+    /// Other create statements.
+    Create {
+        is_create_table_as: bool,
+    },
+    /// Create Materialized views
+    CreateMaterializedView {
+        name: String,
+    },
+    /// Set background ddl
+    SetBackgroundDdl {
+        enable: bool,
+    },
```

Comment:
https://github.com/risingwavelabs/risingwave/pull/13539#discussion_r1400296295

> As SET will be replayed after recovery, is it okay to directly fetch the value of background_ddl with SQL?

If we have `Cmd::SetBackgroundDdl`, we can use it to check if `background_ddl_rate` is set along with manually setting `SET BACKGROUND_DDL`.

Both of these are incompatible, since `background_ddl_rate` can randomly `SET BACKGROUND_DDL`, and make the result of the test with `SET BACKGROUND_DDL` unexpected.

Reply:
IMO it's not much effort to parse it.